cc/ICBT/2022-current.md

```diff
@@ -0,0 +1,43 @@
+# ICBT- International College of Business and Technology
+ 
+Our sole objective is to motivate students in our vibrant community to expand their knowledge in the latest tech industry and to assist them in contributing to open-source projects in the most effective and productive manner.  
+
+## Members
+* Poojith Obeysekara (Club Lead) [@LinkedIn]( https://www.linkedin.com/in/poojithobeysekara/) 
+* Bishmi Dissanayake  
+* Vishaka Randunuge  
+* Lankesh Halangoda
+* Deshan Wickramaarachchi
+* Rukmal Senanayake 
+* Shane Letham
+* Janidu Yapa 
+* Hashan Wickramasooriya 
+* Chanodya Wedagedara
+* Hakeema Ismail
+* Tanushri Welagedara
+* Sivayogaraja Shamridha
+* Sathiyavandhan Sai Shruthika
+* Sachin Peiris
+* Vishal Nirosh
+* Sonali Wegodapola
+* Rasula Rathnayake
+* Rashmi Dissanayaka
+* Ravindu Rajapaksha
+
+
+
+## How to join?
+
+Any student studying within ICBT who’s interested in joining with us , can sign up and join the community as a volunteer.
+- Initially, fill out the Google Form by providing accurate details : https://forms.gle/EPQ4nK4ziW1M8ySQ8
+- Thereafter , you’ll be contacted by one of the Executive Committee Members and will be  added as a member to our WhatsApp Group.
+
+## MIC/Board information
+
+* Master In-Charge – Mr. Induranga De Silva. (mailto: induranga@icbtcampus.edu.lk)
```

Comment:
Change @ and . with (at) and (dot) to prevent basic scrapers from scraping the email.

Reply:
Sure aiya.

Reply:
Thank you